File: app/src/main/java/com/tim/android/service/IViewHandler.java
package com.tim.android.service;

/**
 * IViewHandler
 *
 * @author Tell.Tim
 * @date 2019/12/30 16:42
 */
public interface IViewHandler {
    void onShowQrCode(String qrcode);
    void onShowTimeOut();
    void onShowNetError();
}

File: app/src/main/java/com/tim/android/service/IViewHandler.kt
package com.tim.android.service

/**
 * IViewHandler
 *
 * @author Tell.Tim
 * @date 2019/12/30 16:42
 */
interface IViewHandler {
    fun onShowQrCode(qrcode: String)
    fun onShowTimeOut()
    fun onShowNetError()
    fun onExit()
}
